examples/layout/advanced/mpl_canvas_size.enaml

```diff
@@ -0,0 +1,76 @@
+#------------------------------------------------------------------------------
+# Copyright (c) 2013, Nucleic Development Team.
```

Comment:
Please update the copyright year to 2022

Reply:
Done.